tests/integration/targets/inventory_aws_ec2/playbooks/setup.yml

```diff
@@ -5,7 +5,7 @@
       owner-id: '125523088429'
       virtualization-type: hvm
       root-device-type: ebs
-      name: 'Fedora-Atomic-27*'
+      name: 'Fedora-Cloud-Base-34-1.2.x86_64*'
```

Comment:
Random thought, is there value in standardizing on which image we use?

Reply:
I was annoyed by this performance difference and so I tried a couple of changes. It happens this one don't have any impact on the test execution duration but since the previous image is 5 years older I thought it's better this way.

Reply:
Yeah, I'm just wondering if there's value in us picking an image we use and being a consistent about it, If we did it would probably be worth having a 'helper' which would set an ami_id fact for folks.